modules/ocf_printhost/files/ocfps

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Use the user command pdftops, not the CUPS filter.
+PDFTOPS=/usr/bin/pdftops
+
+if [ "$#" -ne 5 ] && [ "$#" -ne 6 ]; then
+    echo "Usage: $0 job-id user title copies options [file]"
+    exit
+fi
+
+if [ "$#" -eq 5 ]; then
+    $PDFTOPS - - | pstops "$@"
+else
+    $PDFTOPS "$6" - | pstops "${@:1:5}"
+fi
```

Comment:
No, that's fine, it would be good to have documentation on CUPS stuff or anything you had to struggle to find or figure out, that stuff is all good to have, I meant more the specifics that relate to the code itself.

I definitely don't think having either everything in the documentation or everything in the script's comments is ideal though, a mix of the two is probably best, with most stuff in the documentation and anything else in the script perhaps?